packages/lib-core/package.json

```diff
@@ -24,20 +24,30 @@
   "peerDependencies": {
     "lodash-es": "^4.17.21",
     "react": "^17.0.2",
+    "react-redux": "7.2.2",
+    "redux": "4.0.3",
     "yup": "^0.32.11"
   },
   "devDependencies": {
     "@monorepo/common": "0.0.0-fixed",
+    "@openshift/dynamic-plugin-sdk-utils": "0.0.0-alpha1",
     "@rollup/plugin-node-resolve": "^13.1.1",
     "@rollup/plugin-typescript": "^8.3.0",
+    "@types/immutable": "3.x",
     "@types/lodash-es": "^4.17.5",
     "@types/react": "^17.0.37",
+    "@types/react-redux": "6.0.2",
+    "immutable": "3.x",
     "lodash-es": "^4.17.21",
     "react": "^17.0.2",
+    "react-redux": "7.2.2",
+    "redux": "4.0.3",
```

Comment:
@vojtechszocs FYI I needed to bump the version of Redux to `4.0.3` from `4.0.1` (that the console uses).

This is due to a typescript bug fixed in Typescript `4.4.4` (the console is using Typescript `~3.8`).

We will need to keep this in mind when pulling the HAC sdk back into the console.